generated/sample_codegen.rs

```diff
@@ -17794,22 +17794,14 @@ fn Decoder146<'input>(
     let values = ((|| {
         PResult::Ok({
             let mut accum = Vec::new();
-            while _input.remaining() > 0 {
-                let matching_ix = {
-                    _input.open_peek_context();
-                    _input.read_byte()?;
-                    {
-                        let ret = 0;
-                        _input.close_peek_context()?;
-                        ret
+            for n in num_codes.clone() {
```

Comment:
This clone is theoretically necessary since we can't tell in advance whether the Expr-sequence we are iterating over is Copy or can be used referentially, but in this particular case, it might be avoidable in this particular case with specific care...